contracts/locked-astroport-vault/tests/common/compound.rs

```diff
@@ -35,7 +35,7 @@ pub fn test_compound_vault(
     // compound and check that the base token amount corresponding to the users
     // vault token balance has increased.
     let config = robot.query_config();
-    let treasury = config.treasury;
+    let treasury = &config.performance_fee.fee_recipients[0].0;
```

Comment:
add to doc comment that first fee_recipient of performance_fee must be the treasury address

Reply:
I mean this is just to test the balance. Could also call it fee recipient.